examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
Oh I guess it's because none were marked "active" in the DAQ. Can you add a note about what "active" means? Does it mean it can't be used by `ElektaAverager`? Did the user not click a box they should have? etc.